### PR TITLE
 Corrected Faulty Code

### PR DESCRIPTION
**Pull request by AsherAli22**
## Description

[Describe the change that was made in this PR]

### Todos

- [x] Tested and working locally
- [x] Code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] Code changes documented
- [ ] Requested review from >= 2 devs on the team (one frontend and one backend recommended)

### How to test
Updated the code base to remove faulty code. app works now.

### Screenshots and/or Gifs
N/A

### Associated MS Planner Tasks

- [Correct the faulty code in the latest commit](https://tasks.office.com/deakin365.onmicrosoft.com/Home/Task/p4_L9ZwAY06eUYnLCtg99sgACD9h?Type=TaskLink&Channel=Link&CreatedTime=638589454217250000)

### Known Issues

[Describe any known issue with this change]

### CODE REVIEW

File: app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt [hunk 1]
```diff
@@ -54,22 +49,6 @@ class EmailPasswordAuthService(
...
-                    userDataService.updateLastPasswordChangeTimestamp(user.uid, System.currentTimeMillis())
-                    Log.v(logTag, "Password updated successfully for User ${user.uid}")
-                } else {
-                    // Failed to update password
-                    Log.e(logTag, "Failed to update password: ${task.exception}")
-                }
-            }
-        }
-    }
-
```
Comment (Onwa1kenobi) [inline app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt:72]: Is there a reason why we are getting rid of the `updatePassword` function?

File: app/src/main/java/deakin/gopher/guardian/services/UserDataService.kt [hunk 2]
```diff
@@ -27,14 +27,4 @@ class UserDataService : FireStoreDataService() {
...
-    // Amending UserDataService to include last password change timestamp
-    fun updateLastPasswordChangeTimestamp(userId: String, timestamp: Long) {
-        usersCollection.document(userId).update("lastPasswordChange", timestamp)
-            .addOnSuccessListener {
-                Log.v(dataServiceName, "Updated lastPasswordChange for User $userId")
-            }
-            .addOnFailureListener { e ->
-                Log.e(dataServiceName, "Failed to update lastPasswordChange for user $userId: $e")
-            }
-    }
```
Comment (Onwa1kenobi) [inline app/src/main/java/deakin/gopher/guardian/services/UserDataService.kt:39]: Same question here; I was under the impression that this was a feature request. Why are we removing it now?

**Comment by Onwa1kenobi** [comment]
Also @AsherAli22 could you fill out the PR template as well? It will be of no use, if we do not use it. 🙂

File: app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt (hunk 1 above)
Comment (AsherAli22) [reply app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt:72]: currently the faulty codes have been removed before the faulty commit, This was causing issue on the app side not being able to run on user side.

File: app/src/main/java/deakin/gopher/guardian/services/UserDataService.kt (hunk 2 above)
Comment (AsherAli22) [reply app/src/main/java/deakin/gopher/guardian/services/UserDataService.kt:39]: These will be added later with correct logic. Currently these were causing the app not able to build on who ever forks from this repo.

**Review by Onwa1kenobi — APPROVED** [review]

File: app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt (hunk 1 above)
Comment (Onwa1kenobi) [reply app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt:72]: Okay, approving as this entire logic will be migrating to a backend API.

**Pull request merged**